challenge/index.html

```diff
@@ -78,6 +78,29 @@
 
                     </div>
                 </div>
+                <div class="second">
```

Comment:
you have classed and spaced your divs really nicely. I like how they all fit and are are easy to read.